docs/docs/recipes.md

```diff
@@ -2,7 +2,102 @@
 title: Recipes
 ---
 
-This is a stub. Help our community expand it.
+<!-- Basic template:
+Task to accomplish.
+1-2 sentences about it.
+Relevant links out (tutorial, doc pages, plugin readmes, etc).
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+Links:
+* tutorial link
+* docs link
+* additional, if needed
+
+And yeah — those three things are exactly what we're thinking. A first step would be to just go through the tutorial and pull out all the basic things we teach there in a condensed form e.g. creating a site, creating a page, linking between pages, etc. -->
+
+Craving a happy medium between doing the [full tutorial](/tutorial/) and crawling the [full docs]((/tutorial/))? Here's a quick guiding reference for how to build things, Gatsby style.
+
+## Table of Contents
+- [Using a starter](#using-a-starter)
+- [Creating pages](#creating-pages)
+- [Linking between pages](#linking-between-pages)
+- [Styling](#styling)
+- [Creating layouts](#creating-layouts)
+- [Deploying](#deploying)
+- [Querying data](#querying-data)
+- [Sourcing data](#sourcing-data)
+- [Transforming data](#transforming-data)
+
+## Using a starter
+
+Starters are boilerplate Gatsby sites maintained officially, or by the community.
+
```

Comment:
I've gotten a few requests from community members to put the content from this doc and the Quick start doc into the Gatsby READme file since many people use CTRL+f to go through the READme to rapidly look for how to do certain tasks.